### PR TITLE
Projects

### PR DESCRIPTION
**Pull request by Protagonistics**
I like it. Push it to staging por favor!

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -1,25 +1,49 @@
...
-so that anyone can join it with consumer-grade, off-the-shelf Wi-Fi hardware
-available at any electronics store. When you join the PittMesh Network, you not
+# META MESH empowers people to build Community Wireless Networks
+
+**What is a Community Wireless Network?** 
+
+People who have a home Internet connection lease their ability to communicate
+with others to centralized ISP's like Comcast, Verizon, and Time Warner. Other people do not have Internet access at all because for them it is a luxury or ISPs do not service their neighborhoods. Two sides of a coin, two problems.
+
+What if we could solve both at the same time?What if residents came together and built their own infrastructure,
```
Comment (colindean) [inline index.md]: > time?What

wat

File: index.md [hunk 2]
```diff
@@ -1,25 +1,49 @@
...
-area. That network is aptly named [**PittMesh**](http://pittmesh.net). We don't
-use expensive equipment to make this happen- in fact we've designed our network
-so that anyone can join it with consumer-grade, off-the-shelf Wi-Fi hardware
-available at any electronics store. When you join the PittMesh Network, you not
+# META MESH empowers people to build Community Wireless Networks
+
+**What is a Community Wireless Network?** 
+
+People who have a home Internet connection lease their ability to communicate
+with others to centralized ISP's like Comcast, Verizon, and Time Warner. Other people do not have Internet access at all because for them it is a luxury or ISPs do not service their neighborhoods. Two sides of a coin, two problems.
```
Comment (colindean) [inline index.md]: ISPs

File: index.md [hunk 3]
```diff
@@ -1,25 +1,49 @@
...
+
+What if we could solve both at the same time?What if residents came together and built their own infrastructure, 
+funded not by a corporation or even the local government but by the users themselves? Meta Mesh thinks they can by using Community Wireless Networks.
```
Comment (colindean) [inline index.md]: Don't use *they* here. Use *we*.

File: index.md [hunk 4]
```diff
@@ -1,25 +1,49 @@
...
+
+Much like an office's network, resources can be available even without Internet 
+access. You don't need Internet access to use your network printer, right? So why should you need Internet access to send a file to your friend three blocks away? You can both put an inexpensive wireless router and antenna on your roofs, walls, chimneys, or even in your windows, configure them the right way, and you without paying an ISP a dime you have your own wireless network.
+
+And what if, by building that wireless network you opened it up, so that anyone could use it for Internet access? After all, you don't get a refund for bandwidth you don't use, so you might as well give it away-- just like your local coffeeshop does. And by setting up these anteannas and giving away that bandwidth, other people benefit who might not have Internet connections at home.
```
Comment (colindean) [inline index.md]: Don't start with *And*.

Comment (colindean) [inline index.md]: You could try &mdash;` in place of the `--` or just put a real long dash in there via charmap.

Or copy paste this: `–`.

File: index.md [hunk 5]
```diff
@@ -1,25 +1,49 @@
...
+
+And what if, by building that wireless network you opened it up, so that anyone could use it for Internet access? After all, you don't get a refund for bandwidth you don't use, so you might as well give it away-- just like your local coffeeshop does. And by setting up these anteannas and giving away that bandwidth, other people benefit who might not have Internet connections at home. 
+
+What if you organized your community to raise funds to build a network of these wireless devices, a Community Wireless Network, that did all this, used inexpensive routers, had no monthly fees, and created a decentralized, reslient network that could be used for work or for fun when things were good and could be used to get Internet access or to communicate locally when a a bad storm came through and knocked out Internet access? That's a Community Wireless Network. And that is what Meta Mesh wants to help you build one.
```
Comment (colindean) [inline index.md]: > reslient

File: index.md [hunk 6]
```diff
@@ -1,25 +1,49 @@
...
+
+Meta Mesh has two primary goals. 
+
+First, to make Community Wireless Networks so simple that almost any non-technical person could do it. 
+This means writing documentation, streamlining solutions, and educating people so that they can build
+their own networks without the need for an IT company. ISP's, while essential for
```
Comment (colindean) [inline index.md]: > ISP's

File: index.md [hunk 7]
```diff
@@ -1,25 +1,49 @@
...
+network that exists parallel to the Internet and will remain up should Internet access go down.
+Meta Mesh does not own PittMesh; users and node hosts own PittMesh. And while Meta Mesh 
+offers services on the network, it has no mandatory centralized control over it.
+
+{:.img}
+![PittMesh](http://i.imgur.com/cW6BdDk.png)
+
+[**PittMesh**](http://pittmesh.net) is a network of routers connected wirelessly. 
+Meta Mesh uses only consumer-grade, off-the-shelf WiFi hardware
+easily available at electronics stores or online. When you use the PittMesh Network, you not
```
Comment (colindean) [inline index.md]: > Network

network

File: projects.md [hunk 8]
```diff
@@ -2,33 +2,44 @@
...
-## Ongoing Projects
+## [PittMesh](http://pittmesh.net) 
 
-**[PittMesh](http://pittmesh.net)** is a community wireless mesh project. Our
-[informational video](https://www.youtube.com/watch?v=OvAT7DTspaM) explains the
-wireless mesh networking and what we hope to achieve with it in Pittsburgh.
+PittMesh is Meta Mesh's Community Wireless Networking project in Pittsburgh, Pennsylvania, USA. Check out our
+[old informational video](https://www.youtube.com/watch?v=OvAT7DTspaM) which explains
+wireless mesh networking and what we hope to achieve with it in Pittsburgh. We will be updating this video soon, but for now, this should answer some basic questions of how the network works. You can see the current locations of wireless nodes at
+www.pittmesh.net.
```
Comment (colindean) [inline projects.md]: link this

File: projects.md [hunk 9]
```diff
@@ -2,33 +2,44 @@
...
-network.
+Development Corporation](http://mwcdc.org/). This PittMesh segment is considered the "main" segment and is also closest to our offices at the Hardware Store, a coworking space in Allentown.
+
+## The Meta Mesh Webstore
+
+There are two varieties of mesh networking devices available today. You can purchase an all-in-one, networking-as-a-service devices that cost hundreds of dollars each and often require and Internet connection and a yearly license. You can also take the DIY route, buy a supported OpenWRT router, flash the firmware, download the packages, and configure each device by hand. Each solution is daunting. You either need significant financing or significant knowledge to accomplish the respective route.
+
+Meta Mesh is combining the simplicity of purchasing a powerful, enterprise-grade device with the flexibility of a OpenWRT-based firmware, preconfigured for ease-of-use. Launching in September, 2015, the Community Wireless Networking Webstore will be the first of its kind, complete with kit-based purchases and a la carte options. Each device will be pre-assembled and pre-configured by Meta Mesh. All you need to do is mount it and plug it in.
```
Comment (colindean) [inline projects.md]: > September, 2015

no ,

File: projects.md [hunk 10]
```diff
@@ -2,33 +2,44 @@
...
+
+A launch date for the webstore will be announced shortly.
```
Comment (colindean) [inline projects.md]: You just gave a launch date in the previous paragraph.

File: projects.md [hunk 11]
```diff
@@ -2,33 +2,44 @@
...
 
+  While PittMesh can be built one node at a time, seed networks spur faster adoption. Seed networks are funded by grants or generous residents who want to accelerate growth of PittMesh in specific areas of the city. Meta Mesh will work with your community to write a grant on behalf of a 501c3 entity. Since Meta Mesh is an limited liability company, we are not eligible to receive grants directly. Meta Mesh will direct you on how to properly equip a local team from your community with the installation tools, skills, and wireless equipment to build your own network and connect it to the greater PittMesh network. While some of these seed networks may be isolated from other parts of the PittMesh network, once online, we actively seek to connect these network segmemts to the main network either physically or temporarily over the Internet. Meta Mesh holds training classes for your community's installation team to prepare them to install, maintain, and expand the network independent of Meta Mesh. If you don't rely on Meta Mesh to manage your network, you further decentralize who owns and controls the network just like decentralizing the infrastructure itself!
```
Comment (colindean) [inline projects.md]: > 501c3

501(c)3, to be pedantic.

Probably should say "recognized 501(c)3 charitable organization".

**Comment by colindean** [comment]
Are you sure that you want this much stuff on the front page? I don't think it's entirely appropriate for there...

**Comment by Protagonistics** [comment]
Yes, I'd like to have all of it on there. I don't think it's too scary. I'll use some analytics to see how long people stay on the page. If they stay for under 30 seconds I'll shorten it up. But, like i said, I asked to get rid of vision because it felt redundant.

**Comment by Protagonistics** [comment]
Please push this to staging.

**Pull request merged**